pkg/context/context.go

```diff
@@ -111,12 +115,10 @@ func (ctx *Context) Setup() error {
 // You should always call `Teardown` if you called `Setup` to free any resources
 // acquired by `Setup`. Check `Do` for more automated management.
 func (ctx *Context) Teardown() error {
-	if ctx.SnapshotRoot != "" {
-		// if the client code provided the unpack directory,
-		// then it is also in charge of the cleanup.
```

Comment:
Hi @marcel-apf, can you please put the above code comment back into the file? It helps to explain why we're doing this...

Reply:
done